util/reggen/clocking.py

```diff
@@ -49,6 +54,16 @@ def _asdict(self) -> Dict[str, object]:
         ret['primary'] = self.primary
         return ret
 
+    def get_base_name(self) -> str:
```

Comment:
Maybe this should be something like `get_clock_base_name`? A `ClockingItem` might just contain a reset, so I think the method name needs to be a bit more specific.

Reply:
yep good point. although combined with the thing above, i should just reference the variable directly.